app/components/Grid/grid.css

```diff
@@ -1,9 +1,16 @@
+.grid {
+  display: flex;
+  gap: var(--spacing-32);
+  justify-content: center;
+  flex-flow: wrap;
+}
+
 .grid-item {
   cursor: pointer;
   text-align: left;
   padding: var(--spacing-32) var(--spacing-32);
-  min-width: var(--spacing-288);
-  min-height: var(--spacing-288);
+  width: var(--spacing-288);
```

Comment:
I think these should be based on viewport width. I generally won't use variables to define content that is not viewport width (e.g. popovers).

Reply:
won't that get hairy with very small screen? Also, do you have specific values? :D